### DISCUSSION TITLE
Match CD to Specific MusicBrainz Release

### DISCUSSION THREAD

**Discussion by trleith** [automatic-ripping-machine/automatic-ripping-machine#846] [Q&A]
Is there a way to get ARM to ONLY match CD Media, USA Releases in MusicBrainz? It's coming up with some silly stuff.

**Comment by 1337-server** [comment]
Sadly not, we tried to be as broad as possible. Narrowing things down would, by default, cause more issues for the majority. But we should try to add a region setting so that users can narrow things down.

We still need to add the search/manual fix for MusicBrainz, but this is way off for now.
